src/Settings/components/reminders/ReminderOptions/hooks/useFetchBani.js

```diff
@@ -16,37 +16,41 @@ const useFetchBani = (setBaniListData, setReminderBaniData, setStateData, parsed
     try {
       const data = await getBaniList(transliterationLanguage);
       setBaniListData(data);
-
+      const existingKeysSet = parsedReminderBanis.map((bani) => bani.key);
+      const baniOptions = data
+        // Filter out keys for which a reminder has not been created. Ensure that the baniList ID is less than 1000, as we do not manage any bani with IDs greater than 1000.
```

Comment:
No change, just needed some clarification.

When we say, we do not manage any bani with IDs greater than 1000, does that mean, we don't show any bani with id greater than 1000, or we do not allow to set reminders for bani with id greater than 1000.

Reply:
This means we don’t display any bani with an ID greater than 1000. Although list items with these IDs exist, they don’t correspond to any bani in our database.